sql/scan.go

```diff
@@ -129,6 +131,8 @@ type scanNode struct {
 	colKind          colKindMap        // map of column kinds for decoding column values
 	row              parser.DTuple     // the rendered row
 	filter           parser.Expr       // filtering expression for rows
+	offset           parser.Datum      // row offset
+	limit            parser.Datum      // row limit
```

Comment:
I was imagining these would have type `int` and be initialized to `0` and `math.MaxInt` respectively if no limit information were provided. This would simplify the testing against the offset/limit in `Next()`.